src/main/scala/cromwell/engine/backend/ExecutionResult.scala

```diff
@@ -9,6 +9,8 @@ import cromwell.engine.CallOutputs
  */
 sealed trait ExecutionResult
 
+case object SuccessfulFinalCall extends ExecutionResult
+
 /**
```

Comment:
Should the existing `SuccessfulExecution` be renamed to reflect that it corresponds to a `BackendCall`?